### DISCUSSION TITLE
docsはrootディレクトリもしくはdocsフォルダに入れるか？

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#16] [00: 未分類]
自分が作業しているCONTRIBUTING.mdは
はりきってdocsフォルダを作ってその中にファイルを入れたのですが
そうではなく、基本的にはrootディレクトリが良いでしょうか？

@kazzyfrog @kagomen @kana1166

![スクリーンショット 2024-03-12 20 31 52](https://github.com/first-contributions-ja/first-contributions-ja/assets/140990100/11a31434-8e56-43c6-9f0d-902eae2d7cef)

**Comment by pss-aileen** [comment]
画像などをmarkdownに含めたくなったら
どこかしらにフォルダを作った方がよいのかな？と思ったのですが
そちらも含める場合はどうしましょうか？
色々みてるとcommentに画像アップして、そのURLを〜ってありましたが、それはどうなんだろう...と思っています。

`docs/images`
的なものがあっても良いのかなと思っていたりします。

こちらは `readme-only` というフォルダがありました
https://github.com/pss-aileen/Contribute-To-This-Project

**Reply by kazzyfrog** [reply]
> docs/images
的なものがあっても良いのかなと思っていたりします。

確かにそうですね！
一旦docsフォルダを作ってみて、何かしら気になったらあとで変更する方向で良いと思いました！

そしたら、CODE OF CONDUCT.mdも、移行したほうがいいですね

**Reply by pss-aileen** [reply]
これもとにissue作ってみました！！

![スクリーンショット 2024-03-12 21 06 09](https://github.com/first-contributions-ja/first-contributions-ja/assets/140990100/d1ebf816-de23-4caf-a9b0-2c002ad4e3ce)
右上の…からRefarence in new issueで...！

#18 


ご確認ください！
手が空いた方、やりたい方アサインしてお願いします！（自分も気が向いたらするかもですが一旦しない）

**Comment by pss-aileen** [comment]
issue化したらとりあえず閉じてみる。不都合があればreopenする。
